test/src/main/java/org/apache/accumulo/test/LargeSplitRowIT.java

```diff
@@ -130,38 +129,35 @@ public void automaticSplitWith250Same() throws Exception {
     // make a table and lower the configure properties
     final String tableName = getUniqueNames(1)[0];
     try (AccumuloClient client = Accumulo.newClient().from(getClientProperties()).build()) {
-      client.tableOperations().create(tableName);
-      client.tableOperations().setProperty(tableName, Property.TABLE_SPLIT_THRESHOLD.getKey(),
-          "10K");
-      client.tableOperations().setProperty(tableName, Property.TABLE_FILE_COMPRESSION_TYPE.getKey(),
-          "none");
-      client.tableOperations().setProperty(tableName,
-          Property.TABLE_FILE_COMPRESSED_BLOCK_SIZE.getKey(), "64");
-      client.tableOperations().setProperty(tableName, Property.TABLE_MAX_END_ROW_SIZE.getKey(),
-          "1000");
-
-      // Create a BatchWriter and key for a table entry that is longer than the allowed size for an
+      Map<String,String> props = new HashMap<>();
```

Comment:
Would `Map.of()` work here? You use it above.

Reply:
Yea it would work here too. I used hashmap.put here so that each property would be on its own line to make things more readable where as the spot that I used `Map.of()` above, it was only a single property. I could use `Map.of()` with these and do custom formatting to make things more readable. I like how concise `Map.of()` is but it does seem to make things less readable when there are multiple keys and values that get pushed to different lines.

Reply:
I'm all in favor of simplicity of user new API methods. I have no issue with what you did here, I was just curious if there was a reason you couldn't do the same thing.

Reply:
Could do:
```java
    // @formatter:off
    Map<String,String> props = Map.of(
      Property.TABLE_SPLIT_THRESHOLD.getKey(), "10K",
      Property.TABLE_FILE_COMPRESSION_TYPE.getKey(), "none",
      Property.TABLE_FILE_COMPRESSED_BLOCK_SIZE.getKey(), "64",
      Property.TABLE_MAX_END_ROW_SIZE.getKey(), "1000"
    );
    // @formatter:on
```
or
```java
    Map<String,
        String> props = Map.of(Property.TABLE_SPLIT_THRESHOLD.getKey(), "10K",
            Property.TABLE_FILE_COMPRESSION_TYPE.getKey(), "none",
            Property.TABLE_FILE_COMPRESSED_BLOCK_SIZE.getKey(), "64",
            Property.TABLE_MAX_END_ROW_SIZE.getKey(), "1000");
```
with default formatting which isn't too unreadable. Do you think either of these would be better?

Reply:
I think they are functionally equivalent. What's there now is no more or less readable than your suggestions above. I was just curious if there was a technical reason why you could not use `Map.of`.